wavefront_django_sdk/middleware.py

```diff
@@ -0,0 +1,353 @@
+import os
```

Comment:
Sort imports alphabetically and you may add space between distinct groups to improve readability